File: src/main/java/com/thesilentnights/easylogin/service/ByPassService.java
package com.thesilentnights.easylogin.service;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

public class ByPassService {

    private static final List<UUID> list = new ArrayList<>();


    public static void addBypass(UUID uuid) {
        list.add(uuid);
    }

    public static boolean isBypassed(UUID uuid) {
        return list.contains(uuid);
    }

    public void removeBypass(UUID uuid) {
        list.remove(uuid);
    }
}

File: src/main/java/com/thesilentnights/easylogin/service/ByPassService.kt
package com.thesilentnights.easylogin.service

import java.util.*

object ByPassService {
    val list: MutableList<UUID> = ArrayList()

    fun addBypass(uuid: UUID) {
        list.add(uuid)
    }

    fun isBypassed(uuid: UUID): Boolean {
        return list.contains(uuid)
    }

    fun removeBypass(uuid: UUID) {
        list.remove(uuid)
    }
}